src/components/Header/Settings/index.js

```diff
@@ -11,12 +13,34 @@ function Settings()
 {
     return(
         <Popover
-            content={<p style={{ color: '#a7425c' }}>Coming soon</p>}
+            content={
+                <div>
+                    <center>
+                        <Typography.Text
+                            style={{
+                                color: '#a7425c'
+                            }}
+                        >
+                            Prediction size:
```

Comment:
It would be cool to show the current prediction size. We open an issue for this if you prefer.